muselog/django.py

```diff
@@ -0,0 +1,61 @@
+import logging
+import datetime
+
+logger = logging.getLogger(__name__)
+
+
+class MuseDjangoRequestLoggingMiddleware(object):
+
+    def __init__(self, get_response):
+        self.get_response = get_response
+        self.logger = logger
+
+    def __call__(self, request):
+        self.process_request(request)
+        response = self.get_response(request)
+        self.process_response(request, response)
+        return response
+
+    def process_request(self, request):
+        request.started_at = datetime.datetime.utcnow().timestamp()
+
+    def process_response(self, request, response):
+        response_status = response.status_code
+        request_time = 1000.0 * datetime.datetime.utcnow().timestamp() - \
+            request.started_at
+        request_ip = self.get_client_ip(request)
+        request_summary = f"{request.method} {request.get_full_path()} ({request_ip})"
+
+        if response_status < 400:
```

Comment:
Can we actually remove this whole if block and always set the log_method for requests to be `debug`? This is part of a new larger strategy to combat high log overhead.